### ISSUE TITLE
Message prompt is not timely

### ISSUE THREAD

**Issue by hanzhiyan588** [RocketChat/Rocket.Chat#22547]
### Description:

By default, the discussion string is prompted for a message only once. Is there a way to keep the message prompted at all times
![image](https://user-images.githubusercontent.com/63633319/124220041-c9c4dd00-db2f-11eb-8da8-65269e513df4.png)
![image](https://user-images.githubusercontent.com/63633319/124220066-da755300-db2f-11eb-87aa-8b7a5b7aa351.png)
![image](https://user-images.githubusercontent.com/63633319/124220102-eb25c900-db2f-11eb-9f54-33628dfcecf3.png)

### Expected behavior:

Leave it on by default

### Server Setup Information:

- Version of Rocket.Chat Server: 3.16.0
- Operating System: Ubuntu 20.04
- Deployment Method: Docker-compose
- Number of Running Instances: 1
- DB Replicaset Oplog: 1
- NodeJS Version: 12.22.1
- MongoDB Version: 4.0

### Client Setup Information

- Desktop App or Browser Version: Chrome 90.0.4430.212 (Official Build) (64-bit)
- Operating System: Ubuntu 20.04

**Comment by debdutdeb** [comment]
Hi!! 

Can you please explain the issue a bit more? This seems more like a feature request to me (granted I don't really understand what you're talking about). 

Go back a couple of steps and please explain what you're trying to achieve, slowly and as verbosely as possible.

While you're at it, please do not open another issue here on in any other repository. If deemed necessary, I'll transfer this issue over someplace more appropriate.

I appreciate you taking time into Rocket.Chat this much :D 

Thank you! TTYS :)

**Comment by hanzhiyan588** [comment]
> 你好！！
> 
> 你能再解释一下这个问题吗？这对我来说更像是一个功能请求（当然我真的不明白你在说什么）。
> 
> 返回几个步骤，请尽可能缓慢而详细地解释您要实现的目标。
> 
> 在此期间，请不要在任何其他存储库中打开另一个问题。如果认为有必要，我会将这个问题转移到更合适的地方。
> 
> 我很感谢你花时间进入 Rocket.Chat 这么多 :D
> 
> 谢谢！TTYS :)

Can the little bell default and always be enabled

**Comment by debdutdeb** [comment]
So you want -
- the thread notifications to be enabled by default
- "also send to channel" to be disabled by default

**Comment by hanzhiyan588** [comment]
> 所以你要 -
> 
> * 默认启用的线程通知
> * 默认情况下禁用“也发送到频道”

all enabled

**Comment by debdutdeb** [comment]
Hmm.

Sorry but I don't think it's possible to toggle all thread notifications like that. Notifications get turned on automatically, as soon as you participate (send a message). Or if you think you want to follow the thread, you'll have to manually turn on the respective thread's notification. 

As for making "also send to channel" default, I don't know of any such settings, but that doesn't mean it's non-existent. 

I'll investigate more on both of your questions and let you know ASAP :)

**Comment by hanzhiyan588** [comment]
> 唔。
> 
> 抱歉，我认为不可能像这样切换所有线程通知。只要您参与（发送消息），通知就会自动打开。或者，如果您认为要关注该线程，则必须手动打开相应线程的通知。
> 
> 至于将“也发送到频道”设为默认值，我不知道有任何此类设置，但这并不意味着它不存在。
> 
> 我会就你的两个问题进行更多调查，并尽快让你知道:)

![image](https://user-images.githubusercontent.com/63633319/124262905-33aba980-db65-11eb-8577-2d2bca867e59.png)
Only the first message gets people's attention. I hope all the messages get people's attention

**Comment by inetfuture** [comment]
What we want is when a user "mention all" in a thread, all users in this room automatically follow this thread, so all users can get a notification and unread count, regardless of whether the "also send to channel" is selected. 

Current behaivor is if "also send to channel" is not selected, no one will get a notification or unread count, so basically no one will be notified of this message, this is quite suprising for the message sender IMHO.

**Comment by inetfuture** [comment]
@debdutdeb

**Comment by debdutdeb** [comment]
Hi there, this seems to be more of a feature request than an "issue". Can you please open an issue on our feature-requests repository explaining the feature that you're asking? I can talk to someone who can *possibly* get this onto our roadmap. 

https://github.com/RocketChat/feature-requests

In the meantime, I'm going to close this one since it's not exactly a bug. Feel free to reopen if you think otherwise.

Thanks!!
